compact/hcal/backward_sampF.xml

```diff
@@ -12,6 +12,7 @@
 
   <define>
     <constant name="HcalEndcapN_allSensitive" value="1"/>
+    <constant name="HcalEndcapN_length_new"              value="5.0*HcalEndcapN_length"/>
```

Comment:
What's the point of this?

Reply:
This is necessary to have a correct sampling fraction calculation and benchmarks will use it. Making all layers sensitive is not enough and gives wrong e/h response. The correct way is to make it thicker to include the leakage. We tested this.

Reply:
But what is the point of having an alias `HcalEndcapN_length_new` of the same value as `HcalEndcapN_length`?

Reply:
The point is that it allows it to make the detector 5x thicker. Do you prefer another solution to make it work?

Reply:
Okay, I think I start to get it. Can we come up with more descriptive names.
Would if be okay to rename
`HcalEndcapN_length_new` -> `HcalEndcapN_length`
HcalEndcapN_length` -> `HcalEndcapN_baseline_length` (or `HcalEndcapN_nominal_length`)